recipes-extended/jailhouse/jailhouse_0.10.bb

```diff
@@ -0,0 +1,116 @@
+SUMMARY = "Jailhouse, i.MX fork"
+HOMEPAGE = "https://github.com/siemens/jailhouse"
+SECTION = "jailhouse"
+LICENSE = "GPL-2.0 & BSD-2-Clause"
+
+LIC_FILES_CHKSUM = "file://COPYING;md5=9fa7f895f96bde2d47fd5b7d95b6ba4d \
+                 file://tools/root-cell-config.c.tmpl;beginline=6;endline=33;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/jailhouse/hypercall.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/jailhouse/cell-config.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/arch/arm/asm/jailhouse_hypercall.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/arch/arm64/asm/jailhouse_hypercall.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/arch/x86/asm/jailhouse_hypercall.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://driver/jailhouse.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3"
+
+SRCBRANCH = "imx_4.19.35_1.1.0"
+IMX_JAILHOUSE_SRC ?= "git://source.codeaurora.org/external/imx/imx-jailhouse.git;protocol=ssh"
+
+SRC_URI = "${IMX_JAILHOUSE_SRC};branch=${SRCBRANCH}"
+SRCREV = "e83ab629ccc6f44e4caa05bc51c66d2f86546ee6"
+
+DEPENDS = "virtual/kernel make-native \
+        python-mako-native python-mako \
+        dtc-native "
+
+S ="${WORKDIR}/git"
+
+inherit module pythonnative bash-completion setuptools jailhouse-arch
```

Comment:
@panfir This would not built with current `master` as python2 support is dropped.

Can you please change `pythonnative` and `setuptools` to python3 alternatives? Also please review DEPENDS above as this requires the python3 packages now.

-- andrey